### DISCUSSION TITLE
Framework CSS para usar com React

### DISCUSSION THREAD

**Discussion by jdrgoms** [frontendbr/forum#133] [Perguntas e Respostas]
E aí galera, tudo beleza?

Então, tenho feito algumas pesquisas sobre alguns frameworks e tal para usar numa aplicação de grande porte que está sendo desenvolvido com react.  Teremos muitos components ui para criar, porém fazer tudo do zero não seria viável (tempo). Achei alguns outros além dos velhos conhecidos (bootstrap & foundation) mas ainda não cheguei a uma conclusão. Gostei bastante do [****material-ui****](http://www.material-ui.com/#/), porém o repositório possui muitas issues abertas com a tag [`bug`](https://github.com/callemall/material-ui/issues?q=is%3Aissue+is%3Aopen+label%3ABug). Vocês que  trabalham / trabalharam com React, o que estão utilizando / utilizaram?

**Comment by felipefialho** [comment]
c/c @diessica

**Comment by willycamargo** [comment]
Nós temos um projeto em React e outro em MithrilJS aqui na empresa, mas já lhe adianto que se for tudo bem estruturado o framework CSS não vai interferir no desenvolvimento JS. Como seguimos a [spec do material design](https://www.google.com/design/spec/material-design/introduction.html) então optamos por utilizar o [material design lite](https://getmdl.io/). Acredito que a escolha de um framework é delicada e não se trata apenas de decisões técnicas, mas também de mercado.

Qual o objetivo do produto? Qual guideline que irão seguir? A partir de algumas respostas você irá caminhando para a melhor opção a ser utilizada.

PS: Muitas vezes a escolha de um framework/lib errada te faz perder mais tempo do que "desenvolver do zero".
